src/main/java/com/amazon/dlic/auth/ldap/backend/LDAPAuthorizationBackend.java

```diff
@@ -112,6 +113,7 @@ public class LDAPAuthorizationBackend implements AuthorizationBackend {
     public LDAPAuthorizationBackend(final Settings settings, final Path configPath) {
         this.settings = settings;
         this.skipUsersMatcher = WildcardMatcher.from(settings.getAsList(ConfigConstants.LDAP_AUTHZ_SKIP_USERS));
+        this.excludeRolesMatcher = WildcardMatcher.from(settings.getAsList(ConfigConstants.LDAP_AUTHZ_EXCLUDE_ROLES));
```

Comment:
By default doesn't this need to exclude null values?  I'm not sure if there is a test case that verifies the previous behavior

Reply:
The WildcardMatcher handles null values: https://github.com/opensearch-project/security/blob/main/src/main/java/org/opensearch/security/support/WildcardMatcher.java#L187-L200

If there is no value for this setting then no roles are filtered out and the default behavior is to get all backend roles from the LDAP system. 

When a list is configured in the security config. i.e.

```
...
authz:
      ldap_roles:
        description: "Authorize using LDAP"
        http_enabled: true
        authorization_backend:
          type: ldap
          config:
            ...
            skip_users:
              - admin
              - kibanaserver
              - null # Try to configure null here
```

You can specify `null` in the list, but when its read in it will be converted to the string "null"

Reply:
Surprisingly, its possible to add `null` to the list and its read in as null instead of the string "null". i.e. `"skip_users":["admin","kibanaserver","user3",null,"user4"]`

I updated the WildcardMatcher to handle null values in a list and not only cases where the list object is null. For null values in a list, they will be ignored. I chose to update WildcardMatcher instead of filtering null values out of the list so that every instance of using WildcardMatcher will be able to handle null values robustly.